### DISCUSSION TITLE
Hide private roads and paths?

### DISCUSSION THREAD

**Discussion by kversch** [osmandapp/OsmAnd#18025] [Maps Related]
Is it possible to hide private roads or create a different style for them? This is especially usefull when navigating on foot or by bike to hide roads and paths that are private property. Any help would be appreciated

**Comment by DmitryAlexei** [comment]
In *Menu > configure map > details* you can enable *Show access restrictions and toll* (https://osmand.net/docs/user/map/vector-maps#details)
private roads will be rendered according to OsmAnd legend (https://osmand.net/docs/user/map-legend/osmand#access) with red dots

**Reply by kversch** [reply]
I missed that. Of course!
